document/core/appendix/properties.rst

```diff
@@ -745,10 +745,23 @@ In other words, every thread in a valid configuration either runs forever, traps
 Consequently, given a :ref:`valid store <valid-store>`, no computation defined by :ref:`instantiation <exec-instantiation>` or :ref:`invocation <exec-invocation>` of a valid module can "crash" or otherwise (mis)behave in ways not covered by the :ref:`execution <exec>` semantics given in this specification.
 
 
+.. index:: relaxed memory, ! sequential consistency
+
+Sequential Consistency of Data-Race-Free Programs
```

Comment:
If you remember the corrigendum I sent you (still going through the ACM process), we have to be a little apologetic about SC-DRF - the property we inherit from JS is weaker than the ideal one (which is technically false, because of a corner to do with thin-air).

I should handle writing something about this - basically a non-normative note _somewhere_ (appendix?) to the effect that because of known limitations in our ability to formalise relaxed memory, the model has gaps in it which are subjects of active research.

Reply:
agree